src/pages/Chat/Chat.tsx

```diff
@@ -31,7 +32,7 @@ interface ChatProps {
 }
 
 const GATEWAY =
-  "86d2154575a43f3bf9922d9c52f0a63daca1cf352d57ef2b5027e38bc8d8f272";
+  "86d2154575a43f3bf9922d9c52f0a63daca1cf352d57ef2b5027e38bc8d8f272"; // this should be in env
```

Comment:
put gateway to .env, vite.config and github secrets. If not set then a dummy or default value can be used.